packages/ui/core-components/src/lib/atoms/skeletons/Skeleton.svelte

```diff
@@ -4,9 +4,11 @@
 
 <script>
 	import { fade } from 'svelte/transition';
+
+	export let mt = 2;
```

Comment:
feels like we could create a utility prop here

```js
export let class

<div class="{class} animate-pulse h-full w-full mt-2 mb-4" in:fade|local>
```

Reply:
good call - we can use the `cn` function and the `class` prop pattern from Accordion